inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/InlongClusterServiceImpl.java

```diff
@@ -146,7 +146,7 @@ public Integer saveTag(ClusterTagRequest request, String operator) {
     @Override
     public Integer saveTag(ClusterTagRequest request, UserInfo opInfo) {
         // only the person in charges can query
-        if (!opInfo.getRoles().contains(UserTypeEnum.ADMIN.name())) {
+        if (!opInfo.getAccountType().equals(UserTypeEnum.ADMIN.getCode())) {
```

Comment:
Semantically, account type has less meaning than user role. It is more common to manage permissions based on user roles.
So I think this change is not a good practice.

Reply:
I agree with what you said, but the current practice has not yet completed authentication and authorization based on roles, so first judge according to the current implementation based on AccountType, and then deal with it according to roles after subsequent improvements, or it will become a functional bug

Reply:
This PR only adjusts the newly added implementation to be consistent with the current system practice, and authorization issues according to roles will be handle later @healchow

Reply:
Okay, looking forward to your next PR 😄